Automattic/wp-calypso
# Theme Discovery: Theme Showcase show same themes per row as Collection Carousel

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4407

## Proposed Changes

* Added the same breakpoints the Collection Carousel has.
* Implemented the same number of items per row and breakpoint as the Collection Carousel.

## Testing Instructions



* Open the Calypso Live Link
* Navigate to the LoTS & LiTS with no flags.
* You should see that the LoTS and LiTS now display the same number of themes per row as the Collection Carousel.

Before:
![Captura de pantalla 2023-10-30 a las 12 32 46](https://github.com/Automattic/wp-calypso/assets/1989914/2cd869d3-0915-4749-9540-c952a3ef41bf)

After:
![Captura de pantalla 2023-10-30 a las 12 32 35](https://github.com/Automattic/wp-calypso/assets/1989914/873f4550-a35e-4783-bed6-e21380ff6e2a)


## Pre-merge Checklist
